docs/data-point-definition.md

```diff
@@ -8,6 +8,8 @@ Gro defines a "data point" as a discrete result produced by our API. When using
   'value': 0.131714797651,
   'unit_id': 851,
   'reporting_date': None,
+  'metadata': {},
+  'available_date': None,

```

Comment:
`None` shouldn't be a valid value of `available_date` anymore, right? It should either not appear in the result at all (if it wasn't requested), or it should have a timestamp string as the value?

Reply:
good catch! updated with actual timestamp